api-client/src/main/java/io/split/api/client/api/ApiAttributeClient.java

```diff
@@ -0,0 +1,32 @@
+package io.split.api.client.api;
+
+import io.split.api.client.AttributeClient;
+import io.split.api.dtos.Attribute;
+import io.split.api.dtos.TrafficType;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class ApiAttributeClient implements AttributeClient {
+    public List<Attribute> list(String trafficTypeId) {
+        return new ArrayList<>();
+    }
+
+    public List<Attribute> list(TrafficType trafficType) {
+        return list(trafficType.id());
+    }
+
+    public ApiAttributeClient create(Attribute attribute) throws IllegalArgumentException {
```

Comment:
@HJewkes Question here. Why does it return a result of the same type as the service? Shouldn't it be a Boolean, to know if the action was successful or not, or at least the type of the object being created (I mean: Attribute)?